stylus/cozy-ui/defaults.styl

```diff
@@ -64,7 +64,7 @@ input[type=password]
 
 
 [data-input=radio]
-    @extend $checkbox
+    @extend $form-checkbox
```

Comment:
Actually, I don't use defaults.styl for cozy-settings, I just updated this files because that was throwign an error when I updated variable names. Maybe we doesn't it anymore? /poke @m4dz